encoding/cbor/cbor.go

```diff
@@ -0,0 +1,113 @@
+// Package cbor implements partial encoding/decoding of concise binary object
+// representation (CBOR) described in RFC 8949.
+//
+// This package implements a subset of the specification required to support
```

Comment:
nit: what is meant by specification? does this package implement a subset of the smithy rpcv2-cbor protocol or a subset of CBOR as described in RFC 8949?